src/index.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+//import * as serviceWorker from './serviceWorker';
+import Form from "./Form";
+
+
+function Worker() {
+    const [teams, setTeams] = useState("");
```

Comment:
Small thing, but I'd recommend initialising the state here as an empty array instead of a string. Reason being that you're going to be adding team members to this state (so it'll be array of objects - objects that include each team's information).